frontend/src/metabase/visualizations/echarts/cartesian/option/goal-line.ts

```diff
@@ -61,6 +61,13 @@ export function getGoalLineSeriesOption(
         type: [5, 5],
         width: 2,
       },
+      emphasis: {
+        lineStyle: {
```

Comment:
It would be a bit more convenient if you could reuse the entire `lineStyle` object from line 59